guarddog/cli.py

```diff
@@ -83,11 +91,17 @@ def _verify(path, rules, exclude_rules, output_format, exit_non_zero_on_finding,
 
     if output_format == "json":
         import json as js
-        print(js.dumps(results))
+        rv = js.dumps(results)
 
+    if output_format == "sarif":
+        rv = report_verify_sarif(path, list(ALL_RULES), results, ecosystem)
+
+    print(rv)
     if exit_non_zero_on_finding:
         exit_with_status_code(results)
 
+    return rv  # this is mostly for testing
```

Comment:
is it intended that we merge it? (just checking)

Reply:
yes, this is mostly used for end-to-end testing